### DISCUSSION TITLE
Does the latest release always come?

### DISCUSSION THREAD

**Discussion by WindowsHyun** [zelon88/HRConvert2#39] [Q&A]
Currently 3.1 is the latest release.
However, the commit is up to 3.2.8.
In the case of Docker, it is stopped at 2.9.2, so I am using a separate version 3.1.

I would like to try the latest version 3.2.8 or 3.2.7 May I know the release update time?

**Comment by zelon88** [comment]
I have released v3.3, and v3.3.1. 
https://github.com/zelon88/HRConvert2/releases/tag/v3.3

There will be a v3.3.2 hopefully soon. It will enable status indicators for completed file operations.

**Reply by bit-man** [reply]
Only can see release 3.3. Might be I miss something?
